tests/publiccloud/instance_overview.pm

```diff
@@ -56,8 +56,8 @@ sub run {
     }
 
     assert_script_run("rpm -qa > /tmp/rpm.list.txt");
-    upload_logs('/tmp/rpm.list.txt');
-    upload_logs('/var/log/zypper.log');
+    upload_logs('/tmp/rpm.list.txt',   timeout => 180);
```

Comment:
```suggestion
    upload_logs('/tmp/rpm.list.txt',   timeout => 180, failok => 1);
```

I don't see why we should really stop testing here in case any problems